src/main/java/com/github/unldenis/hologram/HologramBuilder.java

```diff
@@ -0,0 +1,83 @@
+package com.github.unldenis.hologram;
+
+import com.github.unldenis.hologram.animation.Animation;
+import com.github.unldenis.hologram.experimental.ClickableTextLine;
+import com.github.unldenis.hologram.line.ILine;
+import com.github.unldenis.hologram.line.ItemLine;
+import com.github.unldenis.hologram.line.Line;
+import com.github.unldenis.hologram.line.TextLine;
+import com.github.unldenis.hologram.line.animated.ItemALine;
+import com.github.unldenis.hologram.line.animated.StandardAnimatedLine;
+import com.github.unldenis.hologram.line.hologram.TextItemStandardLoader;
+import com.github.unldenis.hologram.placeholder.Placeholders;
+import org.bukkit.Location;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.plugin.Plugin;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class HologramBuilder {
```

Comment:
I like the idea of a builder(As it was in previous versions). 
I would add a static method in the Hologram class called builder() that returns a HologramBuilder.